doc/clients/c.rst

```diff
@@ -6,13 +6,25 @@
 Client API
 ==========
 
+The Client API is purpose-built for interaction with RedisAI, 
```

Comment:
Say "the back-end database" instead of Redis so that if/when we add support for Dragon we don't have to change the docs. Please do this everywhere we reference Redis

Reply:
okay will do, good idea and good catch